packages/api-v2/src/AuthProvider.tsx

```diff
@@ -180,23 +180,33 @@ const AuthProvider = ({ loginIDKey, children, cookieTimeout, selectDefaultAccoun
                 setIsAuthorized(false);
                 await mutateAsync({ payload: { authorize: token || '' } })
                     .then(res => {
+                        if (!isMounted) return;
+
                         setIsAuthorized(true);
                         processAuthorizeResponse(res);
                         setIsLoading(false);
                         setIsInitializing(false);
                         setIsSuccess(true);
-                        setLoginid(res?.authorize?.loginid ?? '');
+
+                        // Only set the loginid if the component is still mounted
+                        if (isMounted) {
+                            setLoginid(res?.authorize?.loginid ?? '');
+                        }
```

Comment:
Added isMounted logic for AuthProvider initialize function as well due to memory leak console error which intermittently pops up from DerivAppsGetAccount